Origin: vendor
Subject: Relax dependencies to work well with other rust crates in Debian
Forwarded: not-needed

Tweak env_logger to be the version in Debian.
Hardcode the crypto-nettle backend, as that is used everywhere else in Debian.
Remove env-filter as that doesn't exist in Debian.

diff --git a/Cargo.toml b/Cargo.toml
index cb3035a..1fbb517 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -51,7 +51,7 @@ version = "0.19"
 version = "5"
 
 [dependencies.env_logger]
-version = ">=0.10, <0.12"
+version = "<0.12"
 
 [dependencies.lazy_static]
 version = "1.4.0"
@@ -90,6 +90,7 @@ optional = true
 [dependencies.sequoia-openpgp]
 version = "1.17"
 default-features = false
+features = ["crypto-nettle"]
 
 [dependencies.thiserror]
 version = "1.0.2"
@@ -111,7 +112,7 @@ features = ["compat"]
 version = "0.3"
 
 [dev-dependencies.env_logger]
-version = ">=0.10, <0.12"
+version = "<0.12"
 
 [dev-dependencies.test-log]
 version = "0.2.10"
@@ -122,7 +123,6 @@ default-features = false
 
 [dev-dependencies.tracing-subscriber]
 version = "0.3"
-features = ["env-filter"]
 default-features = false
 
 [build-dependencies.capnpc]
@@ -138,21 +138,5 @@ openpgp-card = ["dep:sequoia-keystore-openpgp-card"]
 softkeys = ["dep:sequoia-keystore-softkeys"]
 tpm = ["dep:sequoia-keystore-tpm"]
 
-[target."cfg(not(windows))".dev-dependencies.sequoia-openpgp]
-version = "1"
-features = [
-    "crypto-nettle",
-    "__implicit-crypto-backend-for-tests",
-]
-default-features = false
-
-[target."cfg(windows)".dev-dependencies.sequoia-openpgp]
-version = "1"
-features = [
-    "crypto-cng",
-    "__implicit-crypto-backend-for-tests",
-]
-default-features = false
-
 [badges.maintenance]
 status = "actively-developed"
